src/muya/lib/index.js

```diff
@@ -45,6 +45,43 @@ class Muya {
     const { focusMode, markdown } = this
     this.setMarkdown(markdown)
     this.setFocusMode(focusMode)
+    this.mutationObserver()
+  }
+
+  mutationObserver () {
+    // Select the node that will be observed for mutations
+    const { container } = this
+
+    // Options for the observer (which mutations to observe)
+    const config = { childList: true, subtree: true }
+
+    // Callback function to execute when mutations are observed
+    const callback = function(mutationsList, observer) {
+      for(const mutation of mutationsList) {
+        if (mutation.type == 'childList') {
+          const { removedNodes, target } = mutation
+          // If the code executes any of the following `if` statements, the editor has gone wrong.
+          // need to report bugs.
+          if (removedNodes && removedNodes.length) {
+            const hasTable = Array.from(removedNodes).some(node => node.nodeType === 1 && node.closest('table.ag-paragraph'))
+            if (hasTable) {
+              console.warn('There was a problem with the table deletion.')
+            }
+          }
+
+          if (target.getAttribute('id') === 'ag-editor-id' && target.childElementCount === 0) {
+            // TODO: the editor can not be input any more. report bugs and recoveryr...
+            console.warn('editor crashed, and can not be input any more.')
```

Comment:
Can we add more information to better reproduce this?

Reply:
It seems that there is no more information now, because we have not recorded the user's operation, etc.